client/my-sites/comments/controller.js

```diff
@@ -9,10 +9,17 @@ import { includes } from 'lodash';
 /**
  * Internal dependencies
  */
-import route from 'lib/route';
 import CommentsManagement from './main';
+import config from 'config';
+import route from 'lib/route';
 
-const VALID_STATUSES = [ 'pending', 'approved', 'spam', 'trash', 'all' ];
+const VALID_STATUSES = [
+	'pending',
+	'approved',
+	'spam',
+	'trash',
+	...config.isEnabled( 'comments/management/all-list' ) && 'all',
```

Comment:
What are the three periods needed for?.. Seems to work without it (_and_ with it).